### ISSUE TITLE
TypeError: 'NoneType' object is not callable

### ISSUE THREAD

**Issue by johnciavarella** [oobabooga/text-generation-webui#1960]
### Describe the bug

Server loads fine and WebUI is function as is changing/loading models. 

When you try to enter a prompt on webui it hangs with a "typing" prompt and spits out this error. This only appeared after running the update.bat today 

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

Reinstalled and the issue is consistent

### Logs

```shell
To create a public link, set `share=True` in `launch()`.
Traceback (most recent call last):
  File "D:\AI_Tools\oobabooga_windows\installer_files\env\lib\site-packages\gradio\routes.py", line 395, in run_predict
    output = await app.get_blocks().process_api(
  File "D:\AI_Tools\oobabooga_windows\installer_files\env\lib\site-packages\gradio\blocks.py", line 1193, in process_api    result = await self.call_function(
  File "D:\AI_Tools\oobabooga_windows\installer_files\env\lib\site-packages\gradio\blocks.py", line 930, in call_function
    prediction = await anyio.to_thread.run_sync(
  File "D:\AI_Tools\oobabooga_windows\installer_files\env\lib\site-packages\anyio\to_thread.py", line 31, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "D:\AI_Tools\oobabooga_windows\installer_files\env\lib\site-packages\anyio\_backends\_asyncio.py", line 937, in run_sync_in_worker_thread
    return await future
  File "D:\AI_Tools\oobabooga_windows\installer_files\env\lib\site-packages\anyio\_backends\_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "D:\AI_Tools\oobabooga_windows\installer_files\env\lib\site-packages\gradio\utils.py", line 491, in async_iteration
    return next(iterator)
  File "D:\AI_Tools\oobabooga_windows\text-generation-webui\modules\chat.py", line 253, in cai_chatbot_wrapper
    for history in chatbot_wrapper(text, state):
  File "D:\AI_Tools\oobabooga_windows\text-generation-webui\modules\chat.py", line 180, in chatbot_wrapper
    prompt = apply_extensions('custom_generate_chat_prompt', text, state, **kwargs)
  File "D:\AI_Tools\oobabooga_windows\text-generation-webui\modules\extensions.py", line 140, in apply_extensions
    return EXTENSION_MAP[typ](*args, **kwargs)
  File "D:\AI_Tools\oobabooga_windows\text-generation-webui\modules\extensions.py", line 81, in _apply_custom_generate_chat_prompt
    return custom_generate_chat_prompt(text, state, **kwargs)
TypeError: 'NoneType' object is not callable
```

### System Info

```shell
Win64bit 80gb Ram 4070TI
```

**Comment by just-someguy** [comment]
Same issue, but I found it's related, as the error implies, to extensions. Didn't test them all, but I can confirm Supoerbooga and Complex_memory cause this issue. Running no extensions except for Gallery works fine. Devs keep uploading things without testing them fully, so I recommend the commit `334486f527bc97f61eb3264def4e03a0dab9b369` or older. If you use `complex_memory` make sure you update it as well since the initial style commit yesterday broke it too, but they quickly fixed it.

**Comment by johnciavarella** [comment]
> Same issue, but I found it's related, as the error implies, to extensions. Didn't test them all, but I can confirm Supoerbooga and Complex_memory cause this issue. Running no extensions except for Gallery works fine. Devs keep uploading things without testing them fully, so I recommend the commit `334486f527bc97f61eb3264def4e03a0dab9b369` or older. If you use `complex_memory` make sure you update it as well since the initial style commit yesterday broke it too, but they quickly fixed it.

Seems like it was the extension flags I set for superbooga/send_pic. 

Interestingly even when I had disabled them via the the UI during prior testing it didn't resolve the issue. I had to remove them from [webui] and bounce it for it to work again.

**Comment by TruthSearchers** [comment]
facing same issue

**Comment by TruthSearchers** [comment]
Solved it just changed the extension code to previous one and it worked

**Comment by johnciavarella** [comment]
fixed in #1965
